src/top.ts

```diff
@@ -46,3 +81,88 @@ export async function topNodes(api: CoreV1Api): Promise<NodeStatus[]> {
     }
     return result;
 }
+
+// Returns the current pod CPU/Memory usage including the CPU/Memory usage of each container
+export async function topPods(api: CoreV1Api, metrics: Metrics, namespace?: string): Promise<PodStatus[]> {
+    // Figure out which pod list endpoint to call
+    const getPodList = async (): Promise<V1PodList> => {
+        if (namespace) {
+            return (await api.listNamespacedPod(namespace)).body;
+        } else {
```

Comment:
nit, no need for the `else { ... } ` just `return (...);`

Reply:
fixed!